picard/ui/scripteditor.py

```diff
@@ -656,6 +656,8 @@ def load(self, reload=False):
         config = get_config()
         self.naming_scripts = config.setting[self.SCRIPTS_LIST_KEY]
         self.selected_script_id = config.setting[self.SELECTED_SCRIPT_KEY]
+        if not self.selected_script_id or self.selected_script_id not in self.naming_scripts:
+            self.selected_script_id = "Preset 1"
```

Comment:
Instead of "Preset 1", I think we should store this in a constant (`DEFAULT_PRESET_ID`?)